spec/requests/v1/users_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Users', type: :request do
+  let(:user) { build(:user) }
+
+  describe 'POST /api/v1/registration' do
+    context 'when valid request' do
+      before do
+        post '/api/v1/registration', params: { email: user.email, password: user.password }
+      end
+
+      it 'creates a new user' do
```

Comment:
it 'renders new user'... Request tests, it's about response codes and bodies. Not about another actions. Also this test should cover all possible response statuses and json schemas matchers for this endpoint.